docs/asciidoc/modules/ROOT/pages/virtual-resource/index.adoc

```diff
@@ -40,7 +42,7 @@ image::apoc.dv.imported-graph-from-RDB.png[scaledwidth="100%"]
 == Managing a Virtualized Resource via JDBC
 
 === Creating a Virtualized Resource (JDBC)
-Before we can query a Virtualized Resource, we need to define it. We do this using the `apoc.dv.catalog.add` procedure.
+Before we can query a Virtualized Resource, we need to define it. We do this using the `apoc.dv.catalog.install` procedure.
 The procedure takes two parameters:
 
 * a name that uniquely identifies the virtualized resource and can be used to query that resource
```

Comment:
```suggestion
The procedure takes three parameters:

* a name that uniquely identifies the virtualized resource and can be used to query that resource
* the database name where we want to use the resource (default is `'neo4j'`)
```

E definire il parametro databaseName anche sul drop, da qualche parte in questo file credo